vuepress/docs/docs/schema.md

```diff
@@ -227,25 +227,42 @@ addSbtPlugin("com.github.ghostdogpr" % "caliban-codegen-sbt" % "1.0.1")
 
 And enable it in your `build.sbt` file:
 ```scala
-enablePlugins(CodegenPlugin)
+enablePlugins(CalibanPlugin)
 ```
 
-Then call the `calibanGenSchema` sbt command.
-```scala
-calibanGenSchema schemaPath outputPath [--scalafmtPath path] [--headers name:value,name2:value2] [--packageName name] [--effect fqdn.Effect] [--scalarMappings gqlType:f.q.d.n.Type,gqlType2:f.q.d.n.Type2] [--imports a.b.c._,c.d.E]
-
-calibanGenSchema project/schema.graphql src/main/MyAPI.scala
-```
-This command will create a Scala file in `outputPath` containing all the types defined in the provided GraphQL schema defined at `schemaPath`. Instead of a file, you can provide a URL and the schema will be obtained using introspection.
+At this point, the `caliban` command will cause any files in `src/main/graphql` to be translated into a Caliban-generated client library. This happens automatically any time you `compile`.
 
-The generated code will be formatted with Scalafmt using the configuration defined by `--scalafmtPath` option (default: `.scalafmt.conf`). If you provide a URL for `schemaPath`, you can provide request headers with `--headers` option.
+By default, all clients are generated with the same client name as the source file, in the `caliban` top-level package.
 
-The package of the generated code is derived from the folder of `outputPath`. This can be overridden by providing an alternative package with the `--packageName` option.
+In order to supply more configuration options to the code generator, you can use the `calibanSettings` sbt setting, combined with the `calibanSetting` function to scope the settings to a particular file:
+```scala
+      // The `file("Service.graphql")` is a path suffix for some file in `src/main/graphql`
+      Compile / caliban / calibanSettings += calibanSetting(file("Service.graphql"))(
+        cs =>
+          cs.packageName("com.example.graphql.client")
+            .scalarMapping(
+              "LanguageCode" -> "com.example.models.LanguageCode",
+            )
+            .scalarMapping(
+              "Timestamp" -> "java.sql.Timestamp",
+              "DayOfWeek" -> "java.time.DayOfWeek",
+              "IntRange"  -> "com.github.tminglei.slickpg.Range[Int]"
+            )
+            .imports("com.example.graphql.client.implicits._")
+      )
+```
 
-By default, each Query and Mutation will be wrapped into a `zio.UIO` effect. This can be overridden by providing an alternative effect with the `--effect` option.
+The settings available on the `cs` (`CalibanSettings`) builder are:
+```
+  def scalafmtPath(path: String): CalibanSettings
+  def headers(pairs: (String,String)*): CalibanSettings
+  def packageName(name: String): CalibanSettings
+  def genView(value: Boolean): CalibanSettings
```

Comment:
This setting doesn't exist on the server codegen.

Reply:
I'm sorry, I don't understand this one. This plugin just runs the client codegen, it seems as though this setting is used there

Reply:
Aaah, I see. I can revert this one